bundles/binding/org.openhab.binding.mpd/src/main/java/org/openhab/binding/mpd/internal/MpdBinding.java

```diff
@@ -219,10 +219,20 @@ private void executePlayerCommand(String playerCommandLine, Object commandParams
                         player.stop();
                         break;
                     case VOLUME_INCREASE:
-                        player.setVolume(player.getVolume() + VOLUME_CHANGE_SIZE);
+                        int result = player.getVolume() + VOLUME_CHANGE_SIZE;
+                        if (volumeInvalid(result)) {
+                            logger.warn("Command ignored because the resulting volume would be invalid: {}", result);
+                        } else {
+                            player.setVolume(result);
+                        }
                         break;
                     case VOLUME_DECREASE:
-                        player.setVolume(player.getVolume() - VOLUME_CHANGE_SIZE);
+                        result = player.getVolume() - VOLUME_CHANGE_SIZE;
+                        if (volumeInvalid(result)) {
+                            logger.warn("Command ignored because the resulting volume would be invalid: {}", result);
+                        } else {
+                            player.setVolume(result);
+                        }
```

Comment:
Doesn't the [player.setVolume() method](https://github.com/msteiger/javampd/blob/master/src/main/java/org/bff/javampd/MPDPlayer.java#L445) already do the same work as the volumeInvalid method, except forces it to be between 0 and 100?  Is a WARN log message necessary?